File: check_images.py
import urllib.request

urls = [
  "https://en.wikipedia.org/wiki/Special:FilePath/Laxman_Jhula_Rishikesh.jpg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Stone_Chariot_at_Vitthala_Temple_in_Hampi.jpg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Araku_Valley.jpg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Tirumala_090615.jpg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Annamalaiyar_Temple_View.jpg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Taj_Mahal_(Edited).jpeg?width=800",
  "https://en.wikipedia.org/wiki/Special:FilePath/Varanasi_Ghats_panorama.jpg?width=800"
]

for u in urls:
    try:
        req = urllib.request.Request(u, headers={'User-Agent': 'Mozilla/5.0'})
        res = urllib.request.urlopen(req)
        print("OK: " + u)
    except Exception as e:
        print("FAIL: " + str(e) + " -> " + u)

File: check_images.js
const https = require('https');
const urls = [
  "https://upload.wikimedia.org/wikipedia/commons/thumb/c/cd/Rishikesh_panorama.jpg/800px-Rishikesh_panorama.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/7/77/Stone_Chariot_at_Vitthala_Temple_in_Hampi.jpg/800px-Stone_Chariot_at_Vitthala_Temple_in_Hampi.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/1/1a/Araku_Valley.jpg/800px-Araku_Valley.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/4/4e/Tirumala_090615.jpg/800px-Tirumala_090615.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/7/75/Annamalaiyar_Temple_View.jpg/800px-Annamalaiyar_Temple_View.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/5/52/Laxman_Jhula.jpg/800px-Laxman_Jhula.jpg",
  "https://upload.wikimedia.org/wikipedia/commons/thumb/8/85/Tourists_at_Araku_valley%2C_Andhra_Pradesh.jpg/800px-Tourists_at_Araku_valley%2C_Andhra_Pradesh.jpg"
];

urls.forEach(u => {
  https.get(u, { headers: { 'User-Agent': 'NodeJS/1.0' } }, (res) => {
    console.log(res.statusCode + " " + u);
  }).on('error', (e) => {
    console.log(e.message + " " + u);
  });
});
